src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,23 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
+    private final Comparator<Employee> employeeComparator = (firstEmployee, secondEmployee) -> {
+        int ageComparison = Integer.compare(firstEmployee.getAge(), secondEmployee.getAge());
+        if (ageComparison != 0) {
+            return ageComparison;
+        }
+        return firstEmployee.getName().compareTo(secondEmployee.getName());
+    };
+
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
```

Comment:
The method name 'getEmployByOrder' might be a typo or could be more descriptive. Consider renaming it to 'getEmployeesByOrder' to accurately reflect that it returns multiple Employee objects and to match the plural form used in the parameter 'employees'.